Makefile

```diff
@@ -33,60 +33,36 @@ pull: ## Pull all images
 	@ make pull-development-images
 	@ make pull-release-images
 
-pull-development-images: ## Pull development images
+pull-development-image: ## Pull development images
```

Comment:
Quick sanity check: if you copy the contents of OSTk Core's Makefile (expect the variable defs at the top) and paste it here, is there any diff left?

Reply:
That's a great idea. I did a quick check and it is now aligned, there were some help string comments and a few command names that were misaligned.
I will make sure the rest of the repo is aligned in a similar fashion.